README.md

```diff
@@ -127,7 +127,7 @@ Limitations:
 * Functions can be executed via both the `/query` and `/mutation` endpoints
 * Conflicting type names in dependencies will be namespaced with their relative path
 * Generic type parameters will be treated as scalars when referenced
-* Deno's `npm:` specifier doesn't currently work with the vendor command
+* Importing multiple versions of the same package using Deno `npm:` package specifiers
```

Comment:
Awesome improvement!

Reply:
Hrm. Maybe I haven't expressed this properly if you think this is awesome 😅. This is supposed to be a limitation.

Reply:
Well the improvement in my mind is that the limitation has been reduced from "NO NPM SUPPORT" to "CONDITIONAL NPM SUPPORT" 😉